### PR TITLE
Rename all instances of "virgin" being used as a variable name

### PR DESCRIPTION
**Pull request by variableundefined**
## What Does This PR Do
As per title, this replaces the inappropriate, and non-self-explanatory term `virgin` which is mostly used to denote a lack of sexual experience, with more context-appropriate terms which is: 
`used`
`modified`
`populated` 

Some coder had way too much fun over 8 years ago. I know, skip the jokes. 

No actual gameplay impact, no changelog.

## Why It's Good For The Game

So that I'm the last future coder to make a double take

## Changelog
:cl:
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
Can relate

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm [hunk 1]
```diff
@@ -65,7 +65,7 @@ Difficulty: Very Hard
 	name = "All Directions"
 	icon_icon = 'icons/effects/effects.dmi'
 	button_icon_state = "at_shield2"
-	chosen_message = "<span class='colossus'>You are now firing in all directions.</span>"
+	chosen_message = "<span class='colosFsus'>You are now firing in all directions.</span>"
```
Comment (AffectedArc07) [inline code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm:68]: What happened here

**Review by AffectedArc07 — APPROVED** [review]
I could make so many jokes here but I will spare you all

**Comment by Spacemanspark** [comment]
When can we expect the baby

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm (hunk 1 above)
Comment (KasparoVy) [reply code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: Sus

**Pull request merged**